README.md

```diff
@@ -114,6 +114,24 @@ Second, you can install in the `databank` directory.
 
 Note that you also need to install and configure your database server.
 
+#### memcached
```

Comment:
Unless we put instructions for _all_ the possible Databank drivers, this doesn't feel like it belongs here.

That being said, this would be awesome in a wiki page.